src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -371,6 +371,11 @@ def create_managed_disk(cmd, resource_group_name, disk_name, location=None,  # p
             subscription=get_subscription_id(cmd.cli_ctx), resource_group=resource_group_name,
             namespace='Microsoft.Compute', type='diskAccesses', name=disk_access)
 
+    if secure_vm_disk_encryption_set is not None and not is_valid_resource_id(secure_vm_disk_encryption_set):
+        secure_vm_disk_encryption_set = resource_id(
+            subscription=get_subscription_id(cmd.cli_ctx), resource_group=resource_group_name,
+            namespace='Microsoft.Compute', type='diskEncryptionSets', name=secure_vm_disk_encryption_set)
```

Comment:
Dose the `secure_vm_disk_encryption_set` parameter also need to support name?

Reply:
Do we need to consider supporting `secure_vm_disk_encryption_se` to pass in name in addition to id (just like `disk_encryption_set`) ?